test-requirements.txt

```diff
@@ -1,32 +1,3 @@
-# Test reqs
-azure-storage==0.36.0
-google-cloud-storage==1.14.0
-h2o==3.22.1.4
-botocore<=1.12.84
-boto3<=1.9.84
-mock==2.0.0
-moto==1.3.7
-pandas<=0.23.4
-prospector[with_pyroma]==0.12.7
-pep8==1.7.1
-pyarrow==0.12.1
-pylint==1.8.2
-pyspark==2.4.0
-pytest==3.2.1
-pytest-cov==2.6.0
-rstcheck==3.2
-scikit-learn==0.20.2
-scipy==1.2.1
-tensorflow==1.12.0
-torch==1.0.1
-torchvision==0.2.2
-pysftp==0.2.9
-keras==2.2.4
-attrdict==2.0.0
-azureml-sdk==1.0.17; python_version >= "3.0"
-cloudpickle==0.8.0
-pytest-localserver==0.5.0
-sqlalchemy==1.3.0
-cryptography==2.5.0
-# test plugin
-tests/resources/mlflow-test-plugin/
+-r travis/small-requirements.txt
```

Comment:
Make `test-requirements.txt` a pointer to `small-requirements.txt`, `large-requirements.txt`, etc. It should still work as it did before / install the same set of dependencies.

Splitting up the requirements files speeds up running small Python tests by avoiding installing large dependencies like TensorFlow & PySpark (these take ~4 min to install).